### ISSUE TITLE
remove dst-extractor from metaprojects (Trac #1812)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1804]
As the release notes say:

```text
trunk
-----
* This project is now obsolete. The extractor modules have been moved to
  the DST project.
```

So, time to remove it from the metaproject externals.  I think it's only icerec and combo.

Migrated from https://code.icecube.wisc.edu/ticket/1812
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:12:38", 
    "description": "As the release notes say:\n{{{\ntrunk\n-----\n* This project is now obsolete. The extractor modules have been moved to\n  the DST project.\n}}}\n\nSo, time to remove it from the metaproject externals.  I think it's only icerec and combo.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067158057333", 
    "component": "other", 
    "summary": "remove dst-extractor from metaprojects", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-08-03T17:17:14", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160803T17:18:04: 
* david.schultz changed component from "icerec" to "other"
* david.schultz commented:

> Actually, I only see it in combo still.

**Comment by icecube-trac** [comment]
Trac update at 20160803T17:36:21: 
* olivas commented:

> In changeset 149038:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="149038"
> removed dst-extractor from externals.  closes #1804.
> ```


* olivas changed resolution from "" to "fixed"
* olivas changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:12:38: 
* olivas commented:

> Milestone August 2016 deleted


* olivas changed milestone from "August 2016" to ""
